docs/data/charts/scatter/ColorScale.js

```diff
@@ -262,6 +262,6 @@ function getGaussianSeriesData(mean, stdev = [0.5, 0.5], N = 50) {
         Math.cos(2.0 * Math.PI * chance.floating({ min: 0, max: 0.99 })) *
         stdev[1] +
       mean[1];
-    return { x, y, z: x + y, id: i };
```

Comment:
> Why adding the mean.join(',') ?

Was a quickfix for having unique ids, the index is already there, adding the mean, which is unique for every call of this function, should make the ids/keys unique overall as well.

Keep in mind these are not in multiple series, they are in a single series, hence why the `key=id=i` was being an issue

> If using indexes causes key to be inconsistent, we could add the seriesId in the key than striking the demo.

Tbh I have no idea why we need the `id`, I suppose it is to prevent duplicate keys? It feels unnecessary though, like we could use an internal id instead of requiring it, but if the user provides it, we should use what they provide.

Reply:
Ok, I did not see that the function was called multiple time to geenrate the data array. I thaugh it it was called once per series 

```
data: [
      ...getGaussianSeriesData([-1, -1]),
      ...getGaussianSeriesData([-1, 1]),
      ...getGaussianSeriesData([1, 1]),
      ...getGaussianSeriesData([1, -1]),
    ]
```